tests/test_fields.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+
+import unittest
+from coaster.utils import LabeledEnum
+from baseframe import __
+from baseframe import baseframe
+import baseframe.forms as forms
+from .fixtures import app1 as app
+
+
+class MY_LENUM(LabeledEnum):
+    FIRST = (1, 'first', __("First"))
+    SECOND = (2, 'second', __("Second"))
+    THIRD = (3, 'third', __("Third"))
+
+
+class MyForm(forms.Form):
+    position = forms.EnumSelectField(__("Position"), lenum=MY_LENUM, default=MY_LENUM.THIRD)
+    position_no_default = forms.EnumSelectField(__("Position Without Default"), lenum=MY_LENUM)
+
+
+class TestFormSQLAlchemy(unittest.TestCase):
+    def setUp(self):
+        baseframe.init_app(app, requires=['baseframe'])
+        self.ctx = app.test_request_context()
+        self.ctx.push()
+        self.form = MyForm(meta={'csrf': False})
+
+    def tearDown(self):
+        self.ctx.pop()
+
+    def test_enumfield(self):
+        self.assertEqual(self.form.position.data, 'third')  # because nor yet validated
```

Comment:
This feels wrong. Regardless of validation it should have the correct value. Look at how `QuerySelectField` does it. We're okay if they have the same problem.